launch/launch/launch_description.py

```diff
@@ -76,6 +76,25 @@ def get_launch_arguments(self, conditional_inclusion=False) -> List[DeclareLaunc
         """
         Return a list of :py:class:`launch.actions.DeclareLaunchArgument` actions.
 
+        See :py:method:`get_launch_arguments_with_launch_descriptions()`
+        for more details.
+        """
+        return [
+            item[0] for item in
+            self.get_launch_arguments_with_include_launch_descriptions(conditional_inclusion)
+        ]
+
+    def get_launch_arguments_with_include_launch_descriptions(
```

Comment:
nitpick: would `get_launch_arguments_with_include_launch_description_actions()` be a better name?

Reply:
fixed in https://github.com/ros2/launch/pull/556/commits/32113b20649a4a64e12ba66526f6c5bc61b7f486